TLM/TLM/UI/Helpers/SerializableUIOptionBase.cs

```diff
@@ -12,7 +12,6 @@ namespace TrafficManager.UI.Helpers {
 
     public abstract class SerializableUIOptionBase<TVal, TUI, TComponent> : ILegacySerializableOption
         where TUI : UIComponent
-        where TVal : IConvertible
```

Comment:
That is required to ensure that `Convert.ChangeType` call in `Value` property will work correctly

Reply:
I see now. I fixed the issue some other way
I do need to drop `IConvertible` because `nullable<bool>` is not `IConvertible`.
Instead I added guard for  `IConvertible`.

It would be nice if `Convert.ChangeType` was type safe so that I would get error. So I added a type safe wrapper.